logql/v2/ast.go

```diff
@@ -114,9 +112,7 @@ func (l *LogFilterExpr) String() string {
 		sb.WriteString(`"`)
 		sb.WriteString(")")
 	} else {
-		sb.WriteString(`"`)
-		sb.WriteString(l.value)
-		sb.WriteString(`"`)
+		sb.WriteString(strconv.Quote(l.value))
```

Comment:
We can apply the same improvement to L110-L112, right?

Reply:
I guess so .. just been conservative to make sure nothing unexpected could break; but doing the same on L110-112, all tests still pass, so I can do that as well

Reply:
Yes please let's use for both case `strconv.Quote` for consistency.